Sources/BraintreeCard/BTCard.swift

```diff
@@ -4,80 +4,107 @@ import Foundation
-
-    /// Optional: the country name associated with the card's billing address.
-    /// - Note: Braintree only accepts specific country names.
-    /// - SeeAlso: https://developer.paypal.com/braintree/docs/reference/general/countries#list-of-countries
-    public var countryName: String?
-
-    /// Optional: the ISO 3166-1 alpha-2 country code specified in the card's billing address.
-    /// - Note: Braintree only accepts specific alpha-2 values.
-    /// - SeeAlso: https://developer.paypal.com/braintree/docs/reference/general/countries#list-of-countries
-    public var countryCodeAlpha2: String?
-
-    /// Optional: the ISO 3166-1 alpha-3 country code specified in the card's billing address.
-    /// - Note: Braintree only accepts specific alpha-3 values.
-    /// - SeeAlso: https://developer.paypal.com/braintree/docs/reference/general/countries#list-of-countries
-    public var countryCodeAlpha3: String?
-
-    ///  Optional: The ISO 3166-1 numeric country code specified in the card's billing address.
-    ///  - Note: Braintree only accepts specific numeric values.
-    /// - SeeAlso: https://developer.paypal.com/braintree/docs/reference/general/countries#list-of-countries
-    public var countryCodeNumeric: String?
-
-    /// Controls whether or not to return validations and/or verification results. By default, this is not enabled.
-    /// - Note: Use this flag with caution. By enabling client-side validation, certain tokenize card requests may result in adding the card to the vault.
-    /// These semantics are not currently documented.
-    public var shouldValidate: Bool = false
-
-    /// Optional: If authentication insight is requested. If this property is set to true, a `merchantAccountID` must be provided. Defaults to false.
-    public var authenticationInsightRequested: Bool = false
-
-    /// Optional: The merchant account ID.
-    public var merchantAccountID: String?
-
+    // MARK: - Internal Properties
+
+    let number: String?
+    let expirationMonth: String?
+    let expirationYear: String?
+    let cvv: String?
+    let postalCode: String?
+    let cardholderName: String?
+    let firstName: String?
+    let lastName: String?
+    let company: String?
+    let streetAddress: String?
+    let extendedAddress: String?
+    let locality: String?
+    let region: String?
+    let countryName: String?
+    let countryCodeAlpha2: String?
+    let countryCodeAlpha3: String?
+    let countryCodeNumeric: String?
+    var shouldValidate: Bool = false
+    var authenticationInsightRequested: Bool = false
+    let merchantAccountID: String?
+
+    // MARK: - Initializer
+    
+    /// Creates a Card
+    /// - Parameters:
+    ///   - number: The card number.
+    ///   - expirationMonth: The expiration month as a one or two-digit number on the Gregorian calendar.
+    ///   - expirationYear:The expiration year as a two or four-digit number on the Gregorian calendar.
+    ///   - cvv: The card verification code (like CVV or CID).
```

Comment:
After confirming with the Cards team, the only required parameters are `number`, `expirationMonth`, `expirationYear`, and `cvv`, the others are optional. Regarding overloading initializers, it’s not necessary, after the latest changes I made

Reply:
Gotcha, does this mean that we no longer support CVV only nonces for validation? Based on this docs string `If you wish to create a CVV-only payment method nonce to verify a card already stored in your Vault, omit all other properties to only collect CVV.` was the intent for an overload. If this is no longer supported we should remove that note since currently this would not be possible.

Reply:
Thanks Jax, for the heads up! I’ve added a new initializer with CVV as the only parameter. As you mentioned, this is only to verify cards that are already stored, let me know what do you think.